### PR TITLE
Silver Graphs

### PR DESCRIPTION
**Pull request by bqi343**
Add / modify tags, move two probs from DFS to flood fill, fix POI Sabotage sol

### CODE REVIEW

File: content/3_Silver/Flood_Fill.mdx [hunk 1]
```diff
@@ -410,12 +468,14 @@ int main() {
 <JavaSection>
 
 <Warning>
-Recursive implementations of floodfill in Java can sometimes lead to StackOverflowError. To avoid Stack Overflow, use your own stack in an iterative implementation like the one below.
+	Recursive implementations of floodfill in Java can sometimes lead to
+	StackOverflowError. To avoid Stack Overflow, use your own stack in an
+	iterative implementation like the one below.
```
Comment (frodakcin) [inline content/3_Silver/Flood_Fill.mdx:473]: Why on separate lines?

File: content/3_Silver/Flood_Fill.mdx [hunk 2]
```diff
@@ -8,45 +8,103 @@ description: 'Finding connected components in a graph represented by a grid.'
...
+			'Snow Boots',
+			'811',
+			'Hard',
+			false,
+			['Flood Fill'],
+			'usaco-811'
+		),
+		new Problem('Silver', 'Multiplayer Moo', '836', 'Hard', false, [
+			'Flood Fill',
+		]),
```
Comment (frodakcin) [inline content/3_Silver/Flood_Fill.mdx:107]: Different format?

File: content/3_Silver/Intro_Tree.mdx [hunk 3]
```diff
@@ -85,10 +93,13 @@ export const problems = {
...
 			false,
-			['Tree', 'DFS'],
+			['Tree'],
 			'ccc-reviews'
 		),
-		new Problem('Silver', 'Clock Tree', '1016', 'Normal', false, ['Tree', 'Bipartite']),
+		new Problem('Silver', 'Clock Tree', '1016', 'Normal', false, [
+			'Tree',
+			'Bipartite',
+		]),
```
Comment (frodakcin) [inline content/3_Silver/Intro_Tree.mdx:102]: Also new format

**Review by frodakcin — APPROVED** [review]
All of it looks good to me. Just noticed some formatting changes though

**Comment by frodakcin** [comment]
(Which I think are OK but I'm just curious why they were changed)

**Comment by bqi343** [comment]
It's just VSCode automatic formatting. Changed the instances you mentioned.

File: content/3_Silver/DFS.mdx [hunk 4]
```diff
@@ -11,37 +11,63 @@ frequency: 4
...
 			'Easy',
 			true,
-			['Tree', 'DFS'],
-			'usaco-788'
+			['Directed Graph'],
+			'usaco-668'
 		),
+		new Problem('Silver', 'Fence Planning', '944', 'Easy', true, [
+			'Connected Components',
+		]),
```
Comment (frodakcin) [inline content/3_Silver/DFS.mdx:45]: oops I missed one. Just scanned through the rest though -- they all seem good.

**Comment by frodakcin** [comment]
All set for me.

**Pull request merged**